src/nxdoc/nuxeo-server/installation/tomcat-server-zip/installing-the-nuxeo-platform-on-linux.md

```diff
@@ -312,152 +312,34 @@ history:
 
-{{{multiexcerpt 'requirements-intro' page='Installation'}}}
-
-Check out the [Installation]({{page page='installation'}})&nbsp;for detailed steps to check your Java version and install it.
-
-## Installing the Nuxeo Platform from the ZIP Archive
-
-Installing the Nuxeo Platform using the ZIP archive only installs the Nuxeo Platform. External dependencies must be installed separately.
-
-**To install the Nuxeo Platform ZIP archive:**
-Unzip the [.zip archive](http://www.nuxeo.com/downloads/) using your favorite tool.
-
-**What's next?**
-Do you want to evaluate the platform? You can now [start the server]({{page page='server-start-and-stop'}}).
-You want to do a complete installation, that is compatible for a production environment? You should [prepare your environment]({{page page='setup-best-practices'}}) now.
-
-## Installing a Nuxeo Virtual Machine Image
-
-{{{multiexcerpt 'install-vm' page='Installing the Nuxeo Platform on Windows'}}}
-
-## Installing the Nuxeo Platform from the APT Repository for Debian and Ubuntu
-
-Installing the Nuxeo Platform using the APT sources for Debian and Ubuntu installs and configures the platform, but it also installs all required dependencies for optimal use of the platform.
-
-{{#> callout type='note' }}
-
-On Debian 8 (jessie), the openjdk-8-jdk is not available in the default repositories. Before installing the Nuxeo package, you will need to:
-*  add the jessie-backports repository (`deb http://httpredir.debian.org/debian jessie-backports main`) to your `/etc/apt/sources.list`
-*  refresh your package list (`apt-get update`)
-*  install the package (`apt-get install -t jessie-backports openjdk-8-jdk`)
-
-{{/callout}}
-
-You can either install the Nuxeo Platform using the OS graphical user interface or from the terminal.
-
-You will need to know two things first:
-
-*   The codename of your distribution (eg **xenial** for Ubuntu 16.04 LTS)
-*   Which kind of Nuxeo release you want to install (Long Term Support, Fast Track or SNAPSHOT; see the page [Nuxeo Release Cycle]({{page space='main' page='nuxeo-release-cycle'}}) for more details).
-
-{{#> callout type='tip' }}
-
-For the examples below, let's say you are using Ubuntu 16.04 LTS ("xenial") and want to install the Nuxeo latest Fast Track release (from the "fasttracks" APT repository; for&nbsp;LTS you would replace "fasttracks" with "releases").
-
-{{/callout}}
-
-### Using the Graphical User Interface
-
-&nbsp;
+First, have a look at the detailed steps to [check your Java version]({{page page='tomcat-server-zip'}}#java-check) and install it.
 
-{{! multiexcerpt name='apt-repo-install-UI'}} {{#> callout type='note' }}
+## Installing the Nuxeo Platform From the ZIP Archive
 
-This requires X11.
+### Install the Nuxeo Platform
 
-{{/callout}}
+Download the [ZIP](https://packages.nuxeo.com/repository/maven-public/org/nuxeo/ecm/distribution/nuxeo-server-tomcat/11.4.42/nuxeo-server-tomcat-11.4.42.zip) archive and unzip it using your favorite tool.
```

Comment:
Note for later, if our packages allows the service API for non authenticated users, we could do: https://packages.nuxeo.com/service/rest/v1/search/assets/download?sort=version&repository=maven-public&maven.artifactId=nuxeo-server-tomcat&maven.extension=zip to get the latest build.

Reply:
But we want the latest promoted version in the doc IMO :)